.circleci/ci_deploy.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/sh -e
```

Comment:
You could use this:

https://github.com/omisego/elixir-omg/blob/master/.circleci/config.yml#L295-L309